### DISCUSSION TITLE
Option to start the videos from the beginning

### DISCUSSION THREAD

**Discussion by Giadej** [AndreyPavlenko/Fermata#56] [Ideas]
Is there any way or option to get the videos to start from the beginning? I have a Playlist of music videos and, unfortunately, when one song ends and the next one starts playing, if it was previously started, it does not start from the beginning but from the point where it was previously interrupted. This involves returning the video to the beginning via the progress bar.

**Comment by AndreyPavlenko** [comment]
To be implemented in the next release.

**Reply by Giadej** [reply]
Thank you @AndreyPavlenko
